onnxruntime/python/tools/tensorrt/perf/build/Dockerfile.custom-trt-perf

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+
+ARG IMAGE=onnxruntime
+FROM ${IMAGE}
+
+ARG ONNXRUNTIME_REPO=https://github.com/Microsoft/onnxruntime
+ARG ONNXRUNTIME_BRANCH=master
+ARG TRT_VERSION=8.0.1.6
+
+SHELL ["/bin/bash", "-c"]
+
+# Prepare onnxruntime repository & build onnxruntime with TensorRT
+RUN git clone --single-branch --branch ${ONNXRUNTIME_BRANCH} --recursive ${ONNXRUNTIME_REPO} onnxruntime &&\
+    /bin/sh onnxruntime/dockerfiles/scripts/install_common_deps.sh &&\
+    cp onnxruntime/docs/Privacy.md /code/Privacy.md &&\
+    cp onnxruntime/dockerfiles/LICENSE-IMAGE.txt /code/LICENSE-IMAGE.txt &&\
+    cp onnxruntime/ThirdPartyNotices.txt /code/ThirdPartyNotices.txt
+
+# Checkout appropriate TRT_VERSION and build
+RUN cd onnxruntime &&\
+    trt_v=${TRT_VERSION%.*.*} &&\
+    add_args="" &&\
+    if [ ${trt_v} != "8.0" ] ; then add_args="--skip_submodule_sync"; fi &&\
```

Comment:
let's not hardcode for a specific version. i.e. if version changes, you'll have to modify the != "8.0" check ?
i would remove the check alltogether.
you can always add skip_submodule_sync since you are cloning the repo from scratch with --recursive option.